@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -42,7 +42,8 @@ import {
     StyleSetEvaluator,
     Value
 } from "@here/harp-datasource-protocol/index-decoder";
-import { LineGroup, triangulateLine } from "@here/harp-lines";
+import { LineGroup } from "@here/harp-lines/lib/Lines";
```

Comment:
Why not including from index.ts anymore?

Reply:
Because `@here/harp-lines` index.ts depends on whole bunch of stuff required only in main thread.
Without this, all the materials and possibly whole MapView and friends land in decoder bundle.

Reply:
I see. This basically boils down to your previous statement to have a module split based on main thread vs. web worker instead of the arbitrary split we have right now.